scripts/manifest.arm64.yml

```diff
@@ -0,0 +1,33 @@
+---
+manifest:
+  schema-version: 1.0
+version: 1.0.0
+products:
+  opensearch: 
+    opensearch-min: https://artifacts.opensearch.org/snapshots/core/opensearch/1.0.0/opensearch-min-1.0.0-SNAPSHOT-linux-arm64-189.tar.gz
```

Comment:
Considering this string `opensearch-min-1.0.0-SNAPSHOT-linux-arm64-189.tar.gz`

* `opensearch-min` name
* `1.0.0` version
* `SNAPSHOT` build flavor/type?
* `linux` operating system
* `arm64` platform - Where is this naming from? sometimes I've seen it as `aarch64` other times `arm`
* `189` build number - Should this be hardcoded in a manifest?
* `.tar.gz` file extension.

Reply:
Manifest files will change each release/build. The plugins/core will provide the link to get their artifacts from. We would just download them and bundle